Mathlib/CategoryTheory/Sites/EqualizerSheafCondition.lean

```diff
@@ -350,6 +350,22 @@ theorem sheaf_condition : (Presieve.ofArrows X π).IsSheafFor P ↔
 
 end Arrows
 
+/-- The sheaf condition for a single morphism is the same as the canonical fork diagram being
+limiting. -/
+lemma isSheafFor_singleton_iff {F : Cᵒᵖ ⥤ Type*} {X Y : C} {f : X ⟶ Y}
```

Comment:
Can you provide a version where the `PullbackCone` is specialized to `pullback.diagonal`? Thanks.